teamcity: failed tests on master: test/TestDatabaseAccessors

The following tests appear to have failed:

[#553433](https://teamcity.cockroachdb.com/viewLog.html?buildId=553433):


```
--- FAIL: test/TestDatabaseAccessors (5.310s)
leaktest.go:110: Leaked goroutine: goroutine 3809 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*ccResolverWrapper).watcher(0xc420867800)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/resolver_conn_wrapper.go:110 +0x182
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*ccResolverWrapper).start
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/resolver_conn_wrapper.go:96 +0x3f
leaktest.go:110: Leaked goroutine: goroutine 3830 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*ccBalancerWrapper).watcher(0xc42166eb40)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/balancer_conn_wrappers.go:122 +0x14a
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.newCCBalancerWrapper
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/balancer_conn_wrappers.go:113 +0x14c
leaktest.go:110: Leaked goroutine: goroutine 3831 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*addrConn).transportMonitor(0xc421d9e780)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/clientconn.go:1234 +0x235
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*addrConn).connect.func1(0xc421d9e780)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/clientconn.go:837 +0x1b5
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*addrConn).connect
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/clientconn.go:828 +0xe1
leaktest.go:110: Leaked goroutine: goroutine 3922 [select]:
	github.com/cockroachdb/cockroach/pkg/rpc.(*Context).runHeartbeat(0xc4221f2000, 0xc42204f360, 0xc4204184a0, 0xf, 0xc4210f7bc0, 0x0, 0x0)
		/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:576 +0x3f1
	github.com/cockroachdb/cockroach/pkg/rpc.(*Context).GRPCDial.func1.1.1(0x22eb600, 0xc420867860)
		/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:507 +0x74
	github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42049bbb0, 0xc420bd8120, 0xc420867830)
		/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xe9
	created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
		/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad
leaktest.go:110: Leaked goroutine: goroutine 3923 [IO wait]:
	internal/poll.runtime_pollWait(0x7f42bbe96bc0, 0x72, 0xc4222c0970)
		/usr/local/go/src/runtime/netpoll.go:173 +0x57
	internal/poll.(*pollDesc).wait(0xc422652a18, 0x72, 0xffffffffffffff00, 0x22d0b00, 0x2f8e9d8)
		/usr/local/go/src/internal/poll/fd_poll_runtime.go:85 +0x9b
	internal/poll.(*pollDesc).waitRead(0xc422652a18, 0xc421834400, 0x400, 0x400)
		/usr/local/go/src/internal/poll/fd_poll_runtime.go:90 +0x3d
	internal/poll.(*FD).Read(0xc422652a00, 0xc421834400, 0x400, 0x400, 0x0, 0x0, 0x0)
		/usr/local/go/src/internal/poll/fd_unix.go:157 +0x17d
	net.(*netFD).Read(0xc422652a00, 0xc421834400, 0x400, 0x400, 0xc4222c0b50, 0xcb6e82, 0x1f04520)
		/usr/local/go/src/net/fd_unix.go:202 +0x4f
	net.(*conn).Read(0xc4210ba158, 0xc421834400, 0x400, 0x400, 0x0, 0x0, 0x0)
		/usr/local/go/src/net/net.go:176 +0x6a
	crypto/tls.(*block).readFromUntil(0xc420860690, 0x7f42bac0f000, 0xc4210ba158, 0x5, 0xc4210ba158, 0xc42258a4f0)
		/usr/local/go/src/crypto/tls/conn.go:493 +0x96
	crypto/tls.(*Conn).readRecord(0xc421cfa000, 0x20e5317, 0xc421cfa120, 0x0)
		/usr/local/go/src/crypto/tls/conn.go:595 +0xe0
	crypto/tls.(*Conn).Read(0xc421cfa000, 0xc42154c000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
		/usr/local/go/src/crypto/tls/conn.go:1156 +0x100
	bufio.(*Reader).Read(0xc421346420, 0xc420567618, 0x9, 0x9, 0x0, 0xc42080c800, 0xc4222c0e00)
		/usr/local/go/src/bufio/bufio.go:216 +0x238
	io.ReadAtLeast(0x22cb5c0, 0xc421346420, 0xc420567618, 0x9, 0x9, 0x9, 0xc42040f320, 0xc4222c0e30, 0xcdf2de)
		/usr/local/go/src/io/io.go:309 +0x86
	io.ReadFull(0x22cb5c0, 0xc421346420, 0xc420567618, 0x9, 0x9, 0x20e4830, 0x200050100, 0xc40000002b)
		/usr/local/go/src/io/io.go:327 +0x58
	github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.readFrameHeader(0xc420567618, 0x9, 0x9, 0x22cb5c0, 0xc421346420, 0x0, 0x0, 0x0, 0xc42166a0e8)
		/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:237 +0x7b
	github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.(*Framer).ReadFrame(0xc4205675e0, 0xc42288bad0, 0xc42288bad0, 0x0, 0x0)
		/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:492 +0xa4
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).reader(0xc421c1e180)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:1173 +0xe1
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:273 +0xbf1
leaktest.go:110: Leaked goroutine: goroutine 3924 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.loopyWriter(0x22eb540, 0xc42166ec00, 0xc422862a80, 0xc421f4afb8)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:727 +0x368
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client.func3(0xc421c1e180)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:305 +0x5e
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:304 +0xd42
leaktest.go:110: Leaked goroutine: goroutine 3925 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).keepalive(0xc421c1e180)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:1323 +0x137
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:309 +0xdb5
leaktest.go:110: Leaked goroutine: goroutine 3940 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.loopyWriter(0x22eb600, 0xc422862900, 0xc4228628a0, 0xc422162fb8)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:727 +0x368
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Server.func2(0xc421c1e000)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:261 +0x5e
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Server
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:260 +0xe08
leaktest.go:110: Leaked goroutine: goroutine 3941 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Server).keepalive(0xc421c1e000)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:956 +0x23b
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Server
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:264 +0xe2d
leaktest.go:110: Leaked goroutine: goroutine 3942 [IO wait]:
	internal/poll.runtime_pollWait(0x7f42bbe96d60, 0x72, 0xc4227d28a0)
		/usr/local/go/src/runtime/netpoll.go:173 +0x57
	internal/poll.(*pollDesc).wait(0xc420f29d18, 0x72, 0xffffffffffffff00, 0x22d0b00, 0x2f8e9d8)
		/usr/local/go/src/internal/poll/fd_poll_runtime.go:85 +0x9b
	internal/poll.(*pollDesc).waitRead(0xc420f29d18, 0xc4216bd800, 0x400, 0x400)
		/usr/local/go/src/internal/poll/fd_poll_runtime.go:90 +0x3d
	internal/poll.(*FD).Read(0xc420f29d00, 0xc4216bd800, 0x400, 0x400, 0x0, 0x0, 0x0)
		/usr/local/go/src/internal/poll/fd_unix.go:157 +0x17d
	net.(*netFD).Read(0xc420f29d00, 0xc4216bd800, 0x400, 0x400, 0xf, 0xc42040f1b0, 0x0)
		/usr/local/go/src/net/fd_unix.go:202 +0x4f
	net.(*conn).Read(0xc42166a740, 0xc4216bd800, 0x400, 0x400, 0x0, 0x0, 0x0)
		/usr/local/go/src/net/net.go:176 +0x6a
	github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*MuxConn).Read(0xc42068d900, 0xc4216bd800, 0x400, 0x400, 0x1ef99a0, 0x0, 0x7f42bac0f020)
		/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux/cmux.go:218 +0x119
	crypto/tls.(*block).readFromUntil(0xc4208605d0, 0x7f42bac0f020, 0xc42068d900, 0x5, 0xc42068d900, 0xc4227d2af0)
		/usr/local/go/src/crypto/tls/conn.go:493 +0x96
	crypto/tls.(*Conn).readRecord(0xc420a46700, 0x20e5317, 0xc420a46820, 0x7f42bbef26c8)
		/usr/local/go/src/crypto/tls/conn.go:595 +0xe0
	crypto/tls.(*Conn).Read(0xc420a46700, 0xc42149c000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
		/usr/local/go/src/crypto/tls/conn.go:1156 +0x100
	bufio.(*Reader).Read(0xc421346300, 0xc4203bfc38, 0x9, 0x9, 0x0, 0x0, 0x1)
		/usr/local/go/src/bufio/bufio.go:216 +0x238
	io.ReadAtLeast(0x22cb5c0, 0xc421346300, 0xc4203bfc38, 0x9, 0x9, 0x9, 0xcdc6a6, 0xc42288b680, 0x0)
		/usr/local/go/src/io/io.go:309 +0x86
	io.ReadFull(0x22cb5c0, 0xc421346300, 0xc4203bfc38, 0x9, 0x9, 0x0, 0x0, 0x0)
		/usr/local/go/src/io/io.go:327 +0x58
	github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.readFrameHeader(0xc4203bfc38, 0x9, 0x9, 0x22cb5c0, 0xc421346300, 0x0, 0x1000100000000, 0x2b00000031, 0x3100010001)
		/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:237 +0x7b
	github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.(*Framer).ReadFrame(0xc4203bfc00, 0xc4228627b0, 0xc4228627b0, 0x0, 0x0)
		/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:492 +0xa4
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Server).HandleStreams(0xc421c1e000, 0xc4227f8030, 0x20e46c8)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:395 +0x37
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).serveStreams(0xc4220e2000, 0x22fabc0, 0xc421c1e000)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:634 +0xeb
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).handleRawConn.func2()
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:591 +0x3c
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).handleRawConn.func3(0xc422522000, 0xc4220e2000, 0x7f42bbe55428, 0xc421c1e000)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:599 +0x27
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).handleRawConn
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:598 +0x5f9
------- Stdout: -------
W180308 20:12:39.554490 3285 server/status/runtime.go:109  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180308 20:12:39.577376 3285 server/config.go:530  [n?] 1 storage engine initialized
I180308 20:12:39.577410 3285 server/config.go:533  [n?] RocksDB cache size: 128 MiB
I180308 20:12:39.577419 3285 server/config.go:533  [n?] store 0: in-memory, size 0 B
I180308 20:12:39.601814 3285 server/node.go:374  [n?] **** cluster 072a9aa0-842e-4485-b490-71e7343a6f24 has been created
I180308 20:12:39.601853 3285 server/server.go:1079  [n?] **** add additional nodes by specifying --join=127.0.0.1:45421
I180308 20:12:39.602659 3285 storage/store.go:1388  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180308 20:12:39.602714 3285 server/node.go:504  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=4.9 KiB), ranges=1, leases=0, writes=0.00, bytesPerReplica={p10=5043.00 p25=5043.00 p50=5043.00 p75=5043.00 p90=5043.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00}
I180308 20:12:39.602743 3285 server/node.go:352  [n1] node ID 1 initialized
I180308 20:12:39.602801 3285 gossip/gossip.go:332  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:45421" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:0 > 
I180308 20:12:39.602977 3285 storage/stores.go:331  [n1] read 0 node addresses from persistent storage
I180308 20:12:39.603059 3285 server/node.go:645  [n1] connecting to gossip network to verify cluster ID...
I180308 20:12:39.603208 3285 server/node.go:670  [n1] node connected via gossip and verified as part of cluster "072a9aa0-842e-4485-b490-71e7343a6f24"
I180308 20:12:39.603231 3285 server/node.go:446  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180308 20:12:39.603318 3285 server/server.go:1197  [n1] starting https server at 127.0.0.1:39729
I180308 20:12:39.603334 3285 server/server.go:1198  [n1] starting grpc/postgres server at 127.0.0.1:45421
I180308 20:12:39.603347 3285 server/server.go:1199  [n1] advertising CockroachDB node at 127.0.0.1:45421
I180308 20:12:39.625109 3639 storage/replica_command.go:838  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180308 20:12:39.637839 3285 sql/event_log.go:121  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:node}
I180308 20:12:39.645909 3667 storage/replica_command.go:838  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I180308 20:12:39.650586 3656 storage/replica_command.go:838  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180308 20:12:39.658061 3671 storage/replica_command.go:838  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180308 20:12:39.666504 3523 storage/replica_command.go:838  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180308 20:12:39.702562 3295 storage/replica_command.go:838  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180308 20:12:39.707626 3664 storage/replica_command.go:838  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
I180308 20:12:39.709357 3285 sql/event_log.go:121  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:node}
I180308 20:12:39.711778 3692 storage/replica_command.go:838  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180308 20:12:39.717501 3704 storage/replica_command.go:838  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180308 20:12:39.719132 3285 sql/event_log.go:121  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:node}
I180308 20:12:39.726410 3735 storage/replica_command.go:838  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180308 20:12:39.730904 3285 sql/event_log.go:121  [n1] Event: "alter_table", target: 4, info: {TableName:system.public.users Statement:ALTER TABLE system.public.users ADD COLUMN IF NOT EXISTS "isRole" BOOL NOT NULL DEFAULT false User:node MutationID:1 CascadeDroppedViews:[]}
I180308 20:12:39.733378 3725 storage/replica_command.go:838  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180308 20:12:39.734082 3285 sql/lease.go:352  [n1] publish: descID=4 (users) version=2 mtime=2018-03-08 20:12:39.73396642 +0000 UTC
I180308 20:12:39.739337 3285 sql/lease.go:352  [n1] publish: descID=4 (users) version=3 mtime=2018-03-08 20:12:39.739218344 +0000 UTC
I180308 20:12:39.740883 3751 storage/replica_command.go:838  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180308 20:12:39.747680 3285 sql/backfill.go:134  [n1] Running backfill for "users", v=3, m=1
I180308 20:12:39.757927 3708 storage/replica_command.go:838  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180308 20:12:39.763446 3757 storage/replica_command.go:838  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180308 20:12:39.765490 3285 sql/lease.go:352  [n1] publish: descID=4 (users) version=4 mtime=2018-03-08 20:12:39.765373942 +0000 UTC
I180308 20:12:39.769023 3858 storage/replica_command.go:838  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180308 20:12:39.770850 3285 sql/event_log.go:121  [n1] Event: "finish_schema_change", target: 4, info: {MutationID:1}
I180308 20:12:39.772126 3285 sql/lease.go:275  publish (count leases): descID=4 name=users version=3 count=1
I180308 20:12:39.773796 3866 sql/lease.go:803  new lease: 4("users") ver=4:1520540300.826582955,0, refcount=0
I180308 20:12:39.776347 3796 storage/replica_command.go:838  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180308 20:12:39.779972 3822 storage/replica_command.go:838  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180308 20:12:39.783776 3876 storage/replica_command.go:838  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180308 20:12:39.789043 3885 storage/replica_command.go:838  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180308 20:12:39.803667 3285 sql/event_log.go:121  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:gen_random_uuid()::STRING User:node}
I180308 20:12:39.804050 3285 server/server.go:1271  [n1] done ensuring all necessary migrations have run
I180308 20:12:39.804068 3285 server/server.go:1274  [n1] serving sql connections
I180308 20:12:39.808318 3900 sql/event_log.go:121  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:45421} Attrs: Locality: ServerVersion:2.0} ClusterID:072a9aa0-842e-4485-b490-71e7343a6f24 StartedAt:1520539959603215604 LastUp:1520539959603215604}
--- FAIL: test/TestDatabaseAccessors (10.010s)
leaktest.go:110: Leaked goroutine: goroutine 3809 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*ccResolverWrapper).watcher(0xc420867800)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/resolver_conn_wrapper.go:110 +0x182
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*ccResolverWrapper).start
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/resolver_conn_wrapper.go:96 +0x3f
leaktest.go:110: Leaked goroutine: goroutine 3830 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*ccBalancerWrapper).watcher(0xc42166eb40)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/balancer_conn_wrappers.go:122 +0x14a
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.newCCBalancerWrapper
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/balancer_conn_wrappers.go:113 +0x14c
leaktest.go:110: Leaked goroutine: goroutine 3831 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*addrConn).transportMonitor(0xc421d9e780)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/clientconn.go:1234 +0x235
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*addrConn).connect.func1(0xc421d9e780)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/clientconn.go:837 +0x1b5
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*addrConn).connect
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/clientconn.go:828 +0xe1
leaktest.go:110: Leaked goroutine: goroutine 3922 [select]:
	github.com/cockroachdb/cockroach/pkg/rpc.(*Context).runHeartbeat(0xc4221f2000, 0xc42204f360, 0xc4204184a0, 0xf, 0xc4210f7bc0, 0x0, 0x0)
		/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:576 +0x3f1
	github.com/cockroachdb/cockroach/pkg/rpc.(*Context).GRPCDial.func1.1.1(0x22eb600, 0xc420867860)
		/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:507 +0x74
	github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42049bbb0, 0xc420bd8120, 0xc420867830)
		/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xe9
	created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
		/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad
leaktest.go:110: Leaked goroutine: goroutine 3923 [IO wait]:
	internal/poll.runtime_pollWait(0x7f42bbe96bc0, 0x72, 0xc4222c0970)
		/usr/local/go/src/runtime/netpoll.go:173 +0x57
	internal/poll.(*pollDesc).wait(0xc422652a18, 0x72, 0xffffffffffffff00, 0x22d0b00, 0x2f8e9d8)
		/usr/local/go/src/internal/poll/fd_poll_runtime.go:85 +0x9b
	internal/poll.(*pollDesc).waitRead(0xc422652a18, 0xc421834400, 0x400, 0x400)
		/usr/local/go/src/internal/poll/fd_poll_runtime.go:90 +0x3d
	internal/poll.(*FD).Read(0xc422652a00, 0xc421834400, 0x400, 0x400, 0x0, 0x0, 0x0)
		/usr/local/go/src/internal/poll/fd_unix.go:157 +0x17d
	net.(*netFD).Read(0xc422652a00, 0xc421834400, 0x400, 0x400, 0xc4222c0b50, 0xcb6e82, 0x1f04520)
		/usr/local/go/src/net/fd_unix.go:202 +0x4f
	net.(*conn).Read(0xc4210ba158, 0xc421834400, 0x400, 0x400, 0x0, 0x0, 0x0)
		/usr/local/go/src/net/net.go:176 +0x6a
	crypto/tls.(*block).readFromUntil(0xc420860690, 0x7f42bac0f000, 0xc4210ba158, 0x5, 0xc4210ba158, 0xc42258a4f0)
		/usr/local/go/src/crypto/tls/conn.go:493 +0x96
	crypto/tls.(*Conn).readRecord(0xc421cfa000, 0x20e5317, 0xc421cfa120, 0x0)
		/usr/local/go/src/crypto/tls/conn.go:595 +0xe0
	crypto/tls.(*Conn).Read(0xc421cfa000, 0xc42154c000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
		/usr/local/go/src/crypto/tls/conn.go:1156 +0x100
	bufio.(*Reader).Read(0xc421346420, 0xc420567618, 0x9, 0x9, 0x0, 0xc42080c800, 0xc4222c0e00)
		/usr/local/go/src/bufio/bufio.go:216 +0x238
	io.ReadAtLeast(0x22cb5c0, 0xc421346420, 0xc420567618, 0x9, 0x9, 0x9, 0xc42040f320, 0xc4222c0e30, 0xcdf2de)
		/usr/local/go/src/io/io.go:309 +0x86
	io.ReadFull(0x22cb5c0, 0xc421346420, 0xc420567618, 0x9, 0x9, 0x20e4830, 0x200050100, 0xc40000002b)
		/usr/local/go/src/io/io.go:327 +0x58
	github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.readFrameHeader(0xc420567618, 0x9, 0x9, 0x22cb5c0, 0xc421346420, 0x0, 0x0, 0x0, 0xc42166a0e8)
		/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:237 +0x7b
	github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.(*Framer).ReadFrame(0xc4205675e0, 0xc42288bad0, 0xc42288bad0, 0x0, 0x0)
		/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:492 +0xa4
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).reader(0xc421c1e180)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:1173 +0xe1
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:273 +0xbf1
leaktest.go:110: Leaked goroutine: goroutine 3924 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.loopyWriter(0x22eb540, 0xc42166ec00, 0xc422862a80, 0xc421f4afb8)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:727 +0x368
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client.func3(0xc421c1e180)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:305 +0x5e
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:304 +0xd42
leaktest.go:110: Leaked goroutine: goroutine 3925 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).keepalive(0xc421c1e180)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:1323 +0x137
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:309 +0xdb5
leaktest.go:110: Leaked goroutine: goroutine 3940 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.loopyWriter(0x22eb600, 0xc422862900, 0xc4228628a0, 0xc422162fb8)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:727 +0x368
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Server.func2(0xc421c1e000)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:261 +0x5e
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Server
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:260 +0xe08
leaktest.go:110: Leaked goroutine: goroutine 3941 [select]:
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Server).keepalive(0xc421c1e000)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:956 +0x23b
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Server
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:264 +0xe2d
leaktest.go:110: Leaked goroutine: goroutine 3942 [IO wait]:
	internal/poll.runtime_pollWait(0x7f42bbe96d60, 0x72, 0xc4227d28a0)
		/usr/local/go/src/runtime/netpoll.go:173 +0x57
	internal/poll.(*pollDesc).wait(0xc420f29d18, 0x72, 0xffffffffffffff00, 0x22d0b00, 0x2f8e9d8)
		/usr/local/go/src/internal/poll/fd_poll_runtime.go:85 +0x9b
	internal/poll.(*pollDesc).waitRead(0xc420f29d18, 0xc4216bd800, 0x400, 0x400)
		/usr/local/go/src/internal/poll/fd_poll_runtime.go:90 +0x3d
	internal/poll.(*FD).Read(0xc420f29d00, 0xc4216bd800, 0x400, 0x400, 0x0, 0x0, 0x0)
		/usr/local/go/src/internal/poll/fd_unix.go:157 +0x17d
	net.(*netFD).Read(0xc420f29d00, 0xc4216bd800, 0x400, 0x400, 0xf, 0xc42040f1b0, 0x0)
		/usr/local/go/src/net/fd_unix.go:202 +0x4f
	net.(*conn).Read(0xc42166a740, 0xc4216bd800, 0x400, 0x400, 0x0, 0x0, 0x0)
		/usr/local/go/src/net/net.go:176 +0x6a
	github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*MuxConn).Read(0xc42068d900, 0xc4216bd800, 0x400, 0x400, 0x1ef99a0, 0x0, 0x7f42bac0f020)
		/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux/cmux.go:218 +0x119
	crypto/tls.(*block).readFromUntil(0xc4208605d0, 0x7f42bac0f020, 0xc42068d900, 0x5, 0xc42068d900, 0xc4227d2af0)
		/usr/local/go/src/crypto/tls/conn.go:493 +0x96
	crypto/tls.(*Conn).readRecord(0xc420a46700, 0x20e5317, 0xc420a46820, 0x7f42bbef26c8)
		/usr/local/go/src/crypto/tls/conn.go:595 +0xe0
	crypto/tls.(*Conn).Read(0xc420a46700, 0xc42149c000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
		/usr/local/go/src/crypto/tls/conn.go:1156 +0x100
	bufio.(*Reader).Read(0xc421346300, 0xc4203bfc38, 0x9, 0x9, 0x0, 0x0, 0x1)
		/usr/local/go/src/bufio/bufio.go:216 +0x238
	io.ReadAtLeast(0x22cb5c0, 0xc421346300, 0xc4203bfc38, 0x9, 0x9, 0x9, 0xcdc6a6, 0xc42288b680, 0x0)
		/usr/local/go/src/io/io.go:309 +0x86
	io.ReadFull(0x22cb5c0, 0xc421346300, 0xc4203bfc38, 0x9, 0x9, 0x0, 0x0, 0x0)
		/usr/local/go/src/io/io.go:327 +0x58
	github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.readFrameHeader(0xc4203bfc38, 0x9, 0x9, 0x22cb5c0, 0xc421346300, 0x0, 0x1000100000000, 0x2b00000031, 0x3100010001)
		/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:237 +0x7b
	github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.(*Framer).ReadFrame(0xc4203bfc00, 0xc4228627b0, 0xc4228627b0, 0x0, 0x0)
		/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:492 +0xa4
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Server).HandleStreams(0xc421c1e000, 0xc4227f8030, 0x20e46c8)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:395 +0x37
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).serveStreams(0xc4220e2000, 0x22fabc0, 0xc421c1e000)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:634 +0xeb
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).handleRawConn.func2()
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:591 +0x3c
	github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).handleRawConn.func3(0xc422522000, 0xc4220e2000, 0x7f42bbe55428, 0xc421c1e000)
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:599 +0x27
	created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).handleRawConn
		/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:598 +0x5f9
------- Stdout: -------
W180308 20:12:39.554490 3285 server/status/runtime.go:109  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180308 20:12:39.577376 3285 server/config.go:530  [n?] 1 storage engine initialized
I180308 20:12:39.577410 3285 server/config.go:533  [n?] RocksDB cache size: 128 MiB
I180308 20:12:39.577419 3285 server/config.go:533  [n?] store 0: in-memory, size 0 B
I180308 20:12:39.601814 3285 server/node.go:374  [n?] **** cluster 072a9aa0-842e-4485-b490-71e7343a6f24 has been created
I180308 20:12:39.601853 3285 server/server.go:1079  [n?] **** add additional nodes by specifying --join=127.0.0.1:45421
I180308 20:12:39.602659 3285 storage/store.go:1388  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180308 20:12:39.602714 3285 server/node.go:504  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=4.9 KiB), ranges=1, leases=0, writes=0.00, bytesPerReplica={p10=5043.00 p25=5043.00 p50=5043.00 p75=5043.00 p90=5043.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00}
I180308 20:12:39.602743 3285 server/node.go:352  [n1] node ID 1 initialized
I180308 20:12:39.602801 3285 gossip/gossip.go:332  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:45421" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:0 > 
I180308 20:12:39.602977 3285 storage/stores.go:331  [n1] read 0 node addresses from persistent storage
I180308 20:12:39.603059 3285 server/node.go:645  [n1] connecting to gossip network to verify cluster ID...
I180308 20:12:39.603208 3285 server/node.go:670  [n1] node connected via gossip and verified as part of cluster "072a9aa0-842e-4485-b490-71e7343a6f24"
I180308 20:12:39.603231 3285 server/node.go:446  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180308 20:12:39.603318 3285 server/server.go:1197  [n1] starting https server at 127.0.0.1:39729
I180308 20:12:39.603334 3285 server/server.go:1198  [n1] starting grpc/postgres server at 127.0.0.1:45421
I180308 20:12:39.603347 3285 server/server.go:1199  [n1] advertising CockroachDB node at 127.0.0.1:45421
I180308 20:12:39.625109 3639 storage/replica_command.go:838  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180308 20:12:39.637839 3285 sql/event_log.go:121  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:node}
I180308 20:12:39.645909 3667 storage/replica_command.go:838  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I180308 20:12:39.650586 3656 storage/replica_command.go:838  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180308 20:12:39.658061 3671 storage/replica_command.go:838  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180308 20:12:39.666504 3523 storage/replica_command.go:838  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180308 20:12:39.702562 3295 storage/replica_command.go:838  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180308 20:12:39.707626 3664 storage/replica_command.go:838  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
I180308 20:12:39.709357 3285 sql/event_log.go:121  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:node}
I180308 20:12:39.711778 3692 storage/replica_command.go:838  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180308 20:12:39.717501 3704 storage/replica_command.go:838  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180308 20:12:39.719132 3285 sql/event_log.go:121  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:node}
I180308 20:12:39.726410 3735 storage/replica_command.go:838  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180308 20:12:39.730904 3285 sql/event_log.go:121  [n1] Event: "alter_table", target: 4, info: {TableName:system.public.users Statement:ALTER TABLE system.public.users ADD COLUMN IF NOT EXISTS "isRole" BOOL NOT NULL DEFAULT false User:node MutationID:1 CascadeDroppedViews:[]}
I180308 20:12:39.733378 3725 storage/replica_command.go:838  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180308 20:12:39.734082 3285 sql/lease.go:352  [n1] publish: descID=4 (users) version=2 mtime=2018-03-08 20:12:39.73396642 +0000 UTC
I180308 20:12:39.739337 3285 sql/lease.go:352  [n1] publish: descID=4 (users) version=3 mtime=2018-03-08 20:12:39.739218344 +0000 UTC
I180308 20:12:39.740883 3751 storage/replica_command.go:838  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180308 20:12:39.747680 3285 sql/backfill.go:134  [n1] Running backfill for "users", v=3, m=1
I180308 20:12:39.757927 3708 storage/replica_command.go:838  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180308 20:12:39.763446 3757 storage/replica_command.go:838  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180308 20:12:39.765490 3285 sql/lease.go:352  [n1] publish: descID=4 (users) version=4 mtime=2018-03-08 20:12:39.765373942 +0000 UTC
I180308 20:12:39.769023 3858 storage/replica_command.go:838  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180308 20:12:39.770850 3285 sql/event_log.go:121  [n1] Event: "finish_schema_change", target: 4, info: {MutationID:1}
I180308 20:12:39.772126 3285 sql/lease.go:275  publish (count leases): descID=4 name=users version=3 count=1
I180308 20:12:39.773796 3866 sql/lease.go:803  new lease: 4("users") ver=4:1520540300.826582955,0, refcount=0
I180308 20:12:39.776347 3796 storage/replica_command.go:838  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180308 20:12:39.779972 3822 storage/replica_command.go:838  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180308 20:12:39.783776 3876 storage/replica_command.go:838  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180308 20:12:39.789043 3885 storage/replica_command.go:838  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180308 20:12:39.803667 3285 sql/event_log.go:121  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:gen_random_uuid()::STRING User:node}
I180308 20:12:39.804050 3285 server/server.go:1271  [n1] done ensuring all necessary migrations have run
I180308 20:12:39.804068 3285 server/server.go:1274  [n1] serving sql connections
I180308 20:12:39.808318 3900 sql/event_log.go:121  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:45421} Attrs: Locality: ServerVersion:2.0} ClusterID:072a9aa0-842e-4485-b490-71e7343a6f24 StartedAt:1520539959603215604 LastUp:1520539959603215604}
------- Stdout: -------
W180308 20:17:31.956036 3741 server/status/runtime.go:109  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180308 20:17:32.016928 3741 server/config.go:530  [n?] 1 storage engine initialized
I180308 20:17:32.017189 3741 server/config.go:533  [n?] RocksDB cache size: 128 MiB
I180308 20:17:32.017307 3741 server/config.go:533  [n?] store 0: in-memory, size 0 B
I180308 20:17:32.160275 3741 server/node.go:374  [n?] **** cluster e211de36-9110-4d9f-af45-2f76ac49dfa3 has been created
I180308 20:17:32.160424 3741 server/server.go:1079  [n?] **** add additional nodes by specifying --join=127.0.0.1:37783
I180308 20:17:32.194486 3741 storage/store.go:1388  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180308 20:17:32.195206 3741 server/node.go:504  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=4.9 KiB), ranges=1, leases=0, writes=0.00, bytesPerReplica={p10=5043.00 p25=5043.00 p50=5043.00 p75=5043.00 p90=5043.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00}
I180308 20:17:32.195528 3741 server/node.go:352  [n1] node ID 1 initialized
I180308 20:17:32.195988 3741 gossip/gossip.go:332  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:37783" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:0 > 
I180308 20:17:32.197138 3741 storage/stores.go:331  [n1] read 0 node addresses from persistent storage
I180308 20:17:32.197729 3741 server/node.go:645  [n1] connecting to gossip network to verify cluster ID...
I180308 20:17:32.218640 3741 server/node.go:670  [n1] node connected via gossip and verified as part of cluster "e211de36-9110-4d9f-af45-2f76ac49dfa3"
I180308 20:17:32.220676 3741 server/node.go:446  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180308 20:17:32.263681 3741 server/server.go:1197  [n1] starting https server at 127.0.0.1:41807
I180308 20:17:32.263825 3741 server/server.go:1198  [n1] starting grpc/postgres server at 127.0.0.1:37783
I180308 20:17:32.263903 3741 server/server.go:1199  [n1] advertising CockroachDB node at 127.0.0.1:37783
I180308 20:17:32.322008 4080 storage/replica_command.go:838  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180308 20:17:32.495224 4123 storage/replica_command.go:838  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
W180308 20:17:32.514215 3805 storage/intent_resolver.go:320  [n1,s1] failed to push during intent resolution: failed to push "sql txn implicit" id=ca15f905 key=/Table/SystemConfigSpan/Start rw=true pri=0.03338996 iso=SERIALIZABLE stat=PENDING epo=0 ts=1520540252.303839349,0 orig=1520540252.303839349,0 max=1520540252.303839349,0 wto=false rop=false seq=7
I180308 20:17:32.540334 3741 sql/event_log.go:121  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:node}
I180308 20:17:32.988503 4091 storage/replica_command.go:838  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
W180308 20:17:33.117297 3819 storage/intent_resolver.go:320  [n1,s1] failed to push during intent resolution: failed to push "split" id=7a9aa5d0 key=/Local/Range/System/NodeLiveness/RangeDescriptor rw=true pri=0.01127417 iso=SERIALIZABLE stat=PENDING epo=0 ts=1520540252.988658508,0 orig=1520540252.988658508,0 max=1520540252.988658508,0 wto=false rop=false seq=3
I180308 20:17:33.163962 4105 storage/replica_command.go:838  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180308 20:17:33.409675 3741 sql/event_log.go:121  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:node}
I180308 20:17:33.499256 3855 storage/replica_command.go:838  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180308 20:17:33.731585 3808 storage/replica_command.go:838  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180308 20:17:33.744081 3741 sql/event_log.go:121  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:node}
I180308 20:17:33.910612 4182 storage/replica_command.go:838  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
I180308 20:17:34.032507 4195 storage/replica_command.go:838  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180308 20:17:34.143215 4189 storage/replica_command.go:838  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180308 20:17:34.229967 4177 storage/replica_command.go:838  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180308 20:17:34.311963 4137 storage/replica_command.go:838  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180308 20:17:34.404325 4139 storage/replica_command.go:838  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180308 20:17:34.491411 3741 sql/event_log.go:121  [n1] Event: "alter_table", target: 4, info: {TableName:system.public.users Statement:ALTER TABLE system.public.users ADD COLUMN IF NOT EXISTS "isRole" BOOL NOT NULL DEFAULT false User:node MutationID:1 CascadeDroppedViews:[]}
I180308 20:17:34.586931 3741 sql/lease.go:352  [n1] publish: descID=4 (users) version=2 mtime=2018-03-08 20:17:34.585735347 +0000 UTC
I180308 20:17:34.594051 4263 storage/replica_command.go:838  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180308 20:17:34.694572 4271 storage/replica_command.go:838  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180308 20:17:34.826606 4141 storage/replica_command.go:838  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180308 20:17:34.834605 3741 sql/lease.go:352  [n1] publish: descID=4 (users) version=3 mtime=2018-03-08 20:17:34.83236724 +0000 UTC
I180308 20:17:34.999883 3741 sql/backfill.go:134  [n1] Running backfill for "users", v=3, m=1
I180308 20:17:35.016532 4256 storage/replica_command.go:838  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180308 20:17:35.152333 4340 storage/replica_command.go:838  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180308 20:17:35.281572 4333 storage/replica_command.go:838  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180308 20:17:35.418970 4386 storage/replica_command.go:838  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180308 20:17:35.425009 3741 sql/lease.go:352  [n1] publish: descID=4 (users) version=4 mtime=2018-03-08 20:17:35.424200332 +0000 UTC
I180308 20:17:35.600099 3741 sql/event_log.go:121  [n1] Event: "finish_schema_change", target: 4, info: {MutationID:1}
I180308 20:17:35.652994 4393 sql/lease.go:803  new lease: 4("users") ver=4:1520540536.828965178,0, refcount=0
I180308 20:17:35.673371 3741 sql/lease.go:275  publish (count leases): descID=4 name=users version=3 count=1
I180308 20:17:35.703071 3741 sql/lease.go:275  publish (count leases): descID=4 name=users version=3 count=1
I180308 20:17:36.399438 3741 sql/event_log.go:121  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:gen_random_uuid()::STRING User:node}
I180308 20:17:36.413672 3741 server/server.go:1271  [n1] done ensuring all necessary migrations have run
I180308 20:17:36.413835 3741 server/server.go:1274  [n1] serving sql connections
I180308 20:17:36.520939 4452 sql/event_log.go:121  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:37783} Attrs: Locality: ServerVersion:2.0} ClusterID:e211de36-9110-4d9f-af45-2f76ac49dfa3 StartedAt:1520540252218834738 LastUp:1520540252218834738}
I180308 20:17:36.577506 3741 util/stop/stopper.go:473  quiescing; tasks left:
1      [async] transport racer
I180308 20:17:36.580245 4085 kv/transport_race.go:67  transport race promotion: ran 32 iterations on up to 815 requests

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/database_test.go
+++ b/pkg/sql/database_test.go
@@ -53,7 +53,6 @@ func TestDatabaseAccessors(t *testing.T) {
 
 	s, _, kvDB := serverutils.StartServer(t, base.TestServerArgs{})
 	defer s.Stopper().Stop(context.TODO())
-	defer leaktest.AfterTest(t)()
 
 	if err := kvDB.Txn(context.TODO(), func(ctx context.Context, txn *client.Txn) error {
 		if _, err := getDatabaseDescByID(ctx, txn, sqlbase.SystemDB.ID); err != nil {

